docs/future-work/cmp-hierarchy-improvements/add-top-down-connections.md

```diff
@@ -2,4 +2,8 @@
 title: Add Top-Down Connections
 ---
 
-One of the main roles of top-down connections is the associative recall and prediction outlined in [Associative Connections](add-associative-connections.md). However, top-down projections can also support decomposing goal-states into specific sub-goals, as discussed in [Decomposing Goal States](../motor-system-improvements/decompose-goals-into-subgoals-communicate.md).
\ No newline at end of file
+In Monty systems, low-level LMs project to high-level LMs, where this projection occurs if their sensory receptive fields are co-aligned. Hierarchical connections should be able to learn a mapping between objects represented at these low-level LMs, and objects represented in the high-level LMs that frequently co-occur. Such learning would be similar to that required for [Generalizing Voting To Associative Connections](../voting-improvements/generalize-voting-to-associative-connections.md).
```

Comment:
I wouldn't compare it to the associative voting connections. The voting connections are between LMs that are modeling the same object. The hierarchical connections are between LMs that model a compositional object and its sub-components. So in the bottom-up connection the lower LMs object ID becomes a feature in the higher level LM. In the top down connection, we associate a feature in the higher level LM with an object in a lower level LM (+ a location). I understand where you are coming from in the sense that this is also just a learned association but maybe its still a bit confusing to read for someone who doesn't have all this background. Maybe we could just remove the reference to the voting plan since it won't be the exact same mechanism.

Reply:
Yeah fair point, thanks